app/src/main/java/com/example/nguyenthanhtungh/myapplication/data/model/Hero.java

```diff
@@ -3,15 +3,15 @@
 public class Hero {
     private int mImageId;
 
-    public Hero(int mImageId) {
-        this.mImageId = mImageId;
+    public Hero(int imageId) {
+        this.mImageId = imageId;
```

Comment:
remove this đi chú.
và research xem vì sao lại sử dụng this trong trường hợp
this.imageId = imageId;

Reply:
khi tham số truyền vào và thuộc tính trùng tên ạ, do nãy em để nó trùng tên nhau, nãy vừa sửa lại tên khác nhau ạ.